melange-core/x86_64-darwin-portability.dylan

```diff
@@ -30,10 +30,11 @@ define constant $default-undefines
   = #["__ENVIRONMENT_MAC_OS_X_VERSION_MIN_REQUIRED__"];
 
 // Set up the search path for .h files
-// cc -E -v /dev/null
+let sdk-root = with-application-output (stream = "xcrun --show-sdk-path")
+    strip(read-to-end(stream))
+  end;
```

Comment:
Please indent according to the [Rectangle Rule](https://github.com/google/google-java-format/wiki/The-Rectangle-Rule).  Housel and I are pretty fond of putting `=` on a new line to reduce indentation.  i.e.,

```dylan
let sdk-root
  = with-application-output(...) end;
```

Reply:
Done, I think!